api/rpc/service/service.proto

```diff
@@ -35,3 +18,25 @@ message PublishSpec {
   uint32 publish_port = 3;
   uint32 internal_port = 4;
 }
+
+message ServiceCreateRequest {
+  ServiceSpec service_spec = 1;
+}
+
+message ServiceCreateResponse {
+  string id = 1;
+}
+
+message RemoveRequest {
+  string ident = 1;
```

Comment:
Sounds fine, it could use a comment to make that clear.

follow up: should RemoveResponse return an ident instead of an id?